### ISSUE TITLE
Chapter 7 - no test data

### ISSUE THREAD

**Issue by Korniluk13** [OpenCVBlueprints/OpenCVBlueprints#24]
When testing method meshwarp from "python/utilites.py", you use some file from the folder "/work/blueprints/c7/data/mesh2.png". Unfortunately, it is not attached to this chapter as data. Therefore user can not test this method with your test data.
 
https://github.com/OpenCVBlueprints/OpenCVBlueprints/blob/master/chapter_7/python/utilities.py#L129

**Comment by StevenPuttemans** [comment]
@liquidmetal is there a chance we can add a mesh as test data to the folder?

**Comment by liquidmetal** [comment]
I'm currently traveling and can't find the original file - but `mesh2.png` should point to a simple image with 9 squares of different colors. However, the central four vertices have been moved around a bit.

Thus, the image contains nine closed polygons. The `meshwarp` takes this set of polygons to the set of nine squares (given the grid lattice positions).

**Comment by Korniluk13** [comment]
@liquidmetal Sorry to bother you, could you say are there any datasets on which the stabilization code was tested? I have suggestions that, for example, the linear integration of a gyroscope is not appropriate, if we talk about a 3D model of camera rotation. Therefore it is necessary to use quaternions or rotation matrices.

**Comment by liquidmetal** [comment]
@Korniluk13 You're correct! :D This project is already quite advanced - introducing quaternions would have been a bit much for what the book aims to be. The linear approximation is good enough for small motions. Using a better rotation model would help achieve better results for larger motion and the optimization would converge faster as well.
